CONTRIBUTING.md

```diff
@@ -30,6 +31,16 @@ The easier it is for us to reproduce it, the faster it'll be fixed!
 
 Please don't include any private/sensitive information in your issue!
 
+## Working On Issues
+
+Once you have decided to contribute to Buildah by working on an issue, check our
+backlog of [open issues](https://github.com/containers/buildah/issues) looking
+for any that do not have an "In Progress" label attached to it.  We often times
```

Comment:
How about
```
Often issues will be assigned to someone, to be worked on at a later time.  If you have the time to work on the issue now add yourself as an assignee, and set the "In Progress" label. 
```

Reply:
@rhatdan I've merged this in with comments from @vrothberg  in the Google Doc and have repushed.